srcpkgs/wezterm/template

```diff
@@ -0,0 +1,67 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220905
+revision=1
+_srcver=20220905-102802-7d4b8249
+wrksrc="${pkgname}-${_srcver}"
+build_style=cargo
+hostmakedepends="pkg-config python3"
+makedepends="nautilus-python fontconfig-devel freetype-devel harfbuzz-devel
+ libssh2-devel libX11-devel libxkbcommon-devel libxkbcommon-x11
+ python3-devel wayland-devel xcb-util-devel xcb-util-image-devel
+ xcb-util-keysyms-devel xcb-util-wm-devel zlib-devel"
+depends="fonts-roboto-ttf"
+short_desc="GPU-accelerated cross-platform terminal emulator and multiplexer"
+maintainer="Joseph Benden <joe@benden.us>"
+license="MIT"
+homepage="https://wezfurlong.org/wezterm/"
+distfiles="https://github.com/wez/wezterm/releases/download/${_srcver}/${pkgname}-${_srcver}-src.tar.gz"
+checksum="5898af2bb2dbedcae2648764d5b7abd3d98b0aa3d05d171b09e0e3f76b7dd545"
+python_version=3
+
+do_check() {
+	local _args="--skip e2e::sftp"
+
+	# cannot use build_style cargo do_check because of --release flag.
+	cargo test --target ${RUST_TARGET} --workspace --locked -- ${_args}
+}
+
+do_install() {
+	vbin target/${RUST_TARGET}/release/wezterm
+	vbin target/${RUST_TARGET}/release/wezterm-gui
+	vbin target/${RUST_TARGET}/release/wezterm-mux-server
+	vbin target/${RUST_TARGET}/release/strip-ansi-escapes
+
+	vinstall assets/shell-integration/wezterm.sh 644 etc/profile.d
+	vinstall assets/icon/terminal.png 644 usr/share/icons/hicolor/128x128/apps org.wezfurlong.wezterm.png
+	vinstall assets/icon/wezterm-icon.svg 644 usr/share/icons/hicolor/scalable/apps org.wezfurlong.wezterm.svg
+	vinstall assets/wezterm.desktop 644 usr/share/applications org.wezfurlong.wezterm.desktop
+	vinstall assets/wezterm.appdata.xml 644 usr/share/metainfo org.wezfurlong.wezterm.appdata.xml
+	vinstall assets/shell-completion/bash 644 usr/share/bash-completion/completions $pkgname
+	vinstall assets/shell-completion/fish 644 usr/share/fish/vendor_completions.d $pkgname.fish
+	vinstall assets/shell-completion/zsh 644 usr/share/zsh/site-functions _$pkgname
+
+	vdoc README.md
+	vlicense LICENSE.md
+}
+
+nautilus-wezterm_package() {
```

Comment:
Many nautilus plugins seem to break with nautilus 43 and will have to be disabled. Could you check the status on this one? If it doesn't work and there isn't a plan to support it in a few weeks, I think we could not add the plugin for now because if we did, we would have to disable it in a few weeks when we merge nautilus 43.

Reply:
The plugin has had recent commits, so it might... I'm thinking it should maybe be removed, no?

Reply:
Removed.